README.md

```diff
@@ -50,7 +52,39 @@ The ingress host names are as follows, where `<base_hostname>` is your deploymen
 * Optional:
   Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
 
-## Deploying
+## Installation
+
+
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
+
+```
+ansible-galaxy collection install redhat.artifact_signer
+```
+
+You can also include it in a requirements.yml file and install it with ansible-galaxy collection install -r requirements.yml, using the format:
```

Comment:
Could we use code formatting for the filenames/commands, rather than regular text?